src/tests/Microsoft.PowerFx.Core.Tests/Microsoft.PowerFx.Core.Tests.csproj

```diff
@@ -55,6 +55,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\..\libraries\Microsoft.PowerFx.Core\Microsoft.PowerFx.Core.csproj" />
+    <ProjectReference Include="..\..\libraries\Microsoft.PowerFx.LanguageServerProtocol\Microsoft.PowerFx.LanguageServerProtocol.csproj" />
```

Comment:
>LanguageServerProtocol [](http://example.com/codeflow?start=65&length=22)

Layering issue... WE should avoid adding this dependency in core. 

we can use .Interpreter Tests for now - it already has the dependency.  (in a pure world, we'd have LSp.Tests)

Reply:
Argh